### PR TITLE
Gangams/monitoring doc changes

### PR DESCRIPTION
**Pull request by ganga1980**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**If applicable**:
- [x] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```release-note
```

### CODE REVIEW

File: docs/kubernetes/monitoring.md [hunk 1]
```diff
@@ -130,9 +130,21 @@ If everything looks ok and Grafana and Influx DB were able to start up, you can
 
 ![Image of Grafana](../images/k8s-monitoring-grafana2.png)
 
-## OMS
+## Azure Monitor for containers
 
-OMS is a hosted monitoring solution by Microsoft. It can be used to aggregate node metrics, logs, and other information. OMS container support can be configured to monitor a kubernetes cluster. For more information checkout the tutorial at Azure Docs [here.](https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/container-service/kubernetes/container-service-tutorial-kubernetes-monitor.md)
+Azure Monitor for containers is a feature designed to monitor the performance of container workloads deployed to AKS-engine (formerly known as ACS-engine) cluster(s)hosted on Azure. Monitoring your containers is critical, especially when you're running a production cluster, at scale, with multiple applications.
```
[the remark this answers is not in the document]
Comment (ganga1980) [reply docs/kubernetes/monitoring.md]: > space between `cluster(s)` and `hosted`

Thanks for finding this :). Fixed.

File: docs/kubernetes/monitoring.md [hunk 2]
```diff
@@ -130,9 +130,21 @@ If everything looks ok and Grafana and Influx DB were able to start up, you can
...
+
+Azure Monitor for containers gives you performance visibility by collecting memory and processor metrics from controllers, nodes, and containers that are available in Kubernetes through the Metrics API. Container logs are also collected. After you enable monitoring from Kubernetes clusters, these metrics and logs are automatically collected for you through a containerized version of the Log Analytics agent for Linux and stored in your Log Analytics workspace.
+
+To setup Azure Monitor for containers for aks-engine cluster, follow the instructions in [azuremonitor-containers](https://github.com/helm/charts/tree/master/incubator/azuremonitor-containers)
+
+Navigate to [azmon-containers](https://aka.ms/azmon-containers) to view the health, metrics and logs of AKS-engine cluster(s).
+
+For more details on how to use the product, see [Azure Monitor for containers](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/container-insights-overview)
+
+Refer [azuremonitor-containers-aks-engine](https://github.com/Microsoft/OMS-docker/blob/aks-engine/README.md) for the supported matrix, troubleshooting and supportability etc.
```
Comment (tariq1890) [inline docs/kubernetes/monitoring.md]: Refer to*

**Comment by tariq1890** [comment]
Thanks for the PR :) . @ganga1980

**Comment by tariq1890** [comment]
LGTM. Requesting review from @jackfrancis

**Pull request merged**